frontend/benefit/applicant/src/components/applications/forms/application/deMinimisAid/extended.ts

```diff
@@ -83,18 +102,12 @@ const useComponent = (): ExtendedComponentProps => {
     return fieldsdef;
   }, [t, fieldNames]);
 
-  const getErrorMessage = (fieldName: string): string | undefined => {
-    // todo: implement error messages
-    // (getIn(formik.touched, fieldName) || isSubmitted) &&
-    // getIn(formik.errors, fieldName)
-    if (isSubmitted && fieldName) {
-      return fieldName;
-    }
-    return '';
-  };
+  const getErrorMessage = (fieldName: string): string | undefined =>
+    getErrorText(formik.errors, formik.touched, fieldName, t, isSubmitted);
```

Comment:
Cannot the return type `string | undefined`  of `getErrorMessage` be inferred?  Does `getErrorText` return `undefined`?

Reply:
yes, forgot to remove after refactoring, good point.